bridges/PicartoBridge.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+class PicartoBridge extends BridgeAbstract
+{
+	const NAME = 'Picarto';
+	const URI = 'https://picarto.tv';
+	const DESCRIPTION = 'Returns the online status of a picarto.tv channel';
+	const MAINTAINER = 'No maintainer';
+	const CACHE_TIMEOUT = 300;// 5min, same as TwitchBridge
+	const PARAMETERS = array(
+		array(
+			'channel' => array(
+				'name' => 'Channel',
+				'type' => 'text',
+				'required' => true,
+				'title' => 'Lowercase channel name as seen in channel URL'
+			)
+		)
+	);
+
+	// See "https://api.picarto.tv/" for Picarto API docs
+	const API_BASE_URI = 'https://api.picarto.tv/api/v1/channel/name/';
+	const TIMEZONE_DEFAULT = 'Europe/Berlin';
+
+	public function collectData()
+	{
+		$channelName = $this->getInput('channel');
+		$apiUrl = $API_BASE_URI() . $channelName;
```

Comment:
Does this line work for you? Looks like a string is used as if it was a function?

Reply:
Right sorry. I think I must have rewritten that line a couple times, would explain this and the semicolon.